config/rbac/gatekeeper_admin_pod_security_policy.yaml

```diff
@@ -0,0 +1,32 @@
+apiVersion: policy/v1beta1
```

Comment:
How come this file is needed? It looks like an exact copy of the one gatekeeper uses and will get created by the operator upon installing gatekeeper.

Reply:
It wasn't needed. Thanks